lib/realms/cards/blob_world.rb

```diff
@@ -6,7 +6,7 @@ def self.key
       end
 
       def execute
-        num = active_player.in_play.select { |c| c.played_this_turn? && c.blob? }.length
+        num = active_player.in_play.cards_in_play.count { |c| c.played_this_turn? && c.blob? }
```

Comment:
maybe `.zone_state` which could have an interface for querying against? Maybe too much?